pkg/management/component.go

```diff
@@ -1450,6 +1453,293 @@ func (c *component) UpdateAuthorizations(ctx context.Context, realmName string,
 	return nil
 }
 
+func (c *component) PutAuthorization(ctx context.Context, realmName string, groupID string, auth api.AuthorizationsRepresentation) error {
+	var accessToken = ctx.Value(cs.CtContextAccessToken).(string)
+
+	group, err := c.keycloakClient.GetGroup(accessToken, realmName, groupID)
+	if err != nil {
+		c.logger.Warn(ctx, "err", err.Error())
+		return err
+	}
+
+	var groupName = *group.Name
+
+	authorizations := api.ConvertToDBAuthorizations(realmName, groupName, auth)
+
+	if err = c.checkAllowedTargetRealmsAndGroupNames(ctx, realmName, authorizations); err != nil {
+		return err
+	}
+
+	if err = ValidateScope(authorizations); err != nil {
+		c.logger.Warn(ctx, "err", err.Error())
+		return errorhandler.CreateBadRequestError(constants.MsgErrInvalidParam + "." + constants.Authorization)
```

Comment:
If errors are more specific (see my comments in authorizationutil.go) in validateScope(s), you just have to return err

Reply:
Corrected in commit 5487d08dc80b6deb27b001c1d24cddb82e05a79c